### DISCUSSION TITLE
Rounded URL/Search bar (Firefox 120 and newer)

### DISCUSSION THREAD

**Discussion by QNetITQ** [QNetITQ/WaveFox#181] [Announcements]
Create a file to add third-party styles (In the options description) and add the code below

```
/* ---------- Rounded URL/Search bar ---------- */

:root
{
    --urlbar-icon-border-radius: 50px !important;
}

#urlbar:not([open]) .urlbar-background, #searchbar
{
    --toolbarbutton-border-radius: 50px !important;
}
```

**Comment by davidbepo** [comment]
this has stopped working on firefox 143 :(

**Reply by QNetITQ** [reply]
Mozilla has changed a lot of code recently. I can't keep up with everything. I think by version 145, this code will become part of the style.

UPD
It should be fine now.
